CRAN-SUBMISSION

```diff
@@ -0,0 +1,3 @@
+Version: 1.2.0
```

Comment:
What does this file do?

Reply:
automated by devtools during CRAN submission. https://devtools.r-lib.org/reference/submit_cran.html